backend/python/tools/insert_test_data.py

```diff
@@ -0,0 +1,341 @@
+                StoryContent(
+                    story_id=story_id,
+                    line_index=i,
+                    content=content,
+                ),
+            )
+
+    # story translations
+    db.engine.execute("ALTER TABLE story_translations AUTO_INCREMENT = 1;")
+    db.session.bulk_save_objects(
+        [
+            StoryTranslation(
+                story_id=1, language="GERMAN", stage="TRANSLATE", translator_id=5
+            ),
+            StoryTranslation(
+                story_id=1,
+                language="ENGLISH_UK",
+                stage="TRANSLATE",
+                translator_id=4,
+            ),
+            StoryTranslation(
+                story_id=2, language="GERMAN", stage="TRANSLATE", translator_id=2
+            ),
+            StoryTranslation(
+                story_id=2, language="POLISH", stage="TRANSLATE", translator_id=4
+            ),
+            StoryTranslation(
+                story_id=3, language="MANDARIN", stage="TRANSLATE", translator_id=5
+            ),
+            StoryTranslation(
+                story_id=3,
+                language="ENGLISH_UK",
+                stage="TRANSLATE",
+                translator_id=4,
+            ),
+            StoryTranslation(
+                story_id=4, language="GERMAN", stage="TRANSLATE", translator_id=6
+            ),
+            StoryTranslation(
+                story_id=4,
+                language="ENGLISH_UK",
+                stage="TRANSLATE",
+                translator_id=4,
+            ),
+            StoryTranslation(
+                story_id=5, language="GERMAN", stage="TRANSLATE", translator_id=2
+            ),
+            StoryTranslation(
+                story_id=5,
+                language="ENGLISH_UK",
+                stage="TRANSLATE",
+                translator_id=1,
+            ),
+            StoryTranslation(
+                story_id=5,
+                language="PORTUGUESE",
+                stage="TRANSLATE",
+                translator_id=7,
+            ),
+            StoryTranslation(
+                story_id=5, language="DUTCH", stage="TRANSLATE", translator_id=3
+            ),
+            StoryTranslation(
+                story_id=6,
+                language="ENGLISH_US",
+                stage="TRANSLATE",
+                translator_id=1,
+            ),
+        ]
+    )
+    db.session().commit()
+
+    # story translation contents
+    db.engine.execute("ALTER TABLE story_translation_contents AUTO_INCREMENT = 1;")
+    full_translation = [2, 3, 5, 7, 11, 13]
+    for story_translation_id in full_translation:
+        for i, content in enumerate(generic_content):
+            db.session.add(
+                StoryTranslationContent(
+                    story_translation_id=story_translation_id,
+                    line_index=i,
+                    translation_content=content,
+                )
+            )
+
+    empty_translation = [1, 4, 6, 8, 9, 10, 12]
```

Comment:
do we want to have story_translation_id=12 have only the last line translated and the rest empty?

Reply:
okay also do not really use it so I'll leave it as is for now; if we need it later I can add it in!